walkeeper/src/receive_wal.rs

```diff
@@ -176,6 +176,7 @@ fn request_callback(conf: WalAcceptorConf, timelineid: ZTimelineId) {
         }
 
         if let Some(period) = conf.recall_period {
+            info!("sleeping for {}", humantime::format_duration(period));
```

Comment:
Is this intentional?

Reply:
Intentional at the beginning, but after your comment I realized that it is redundant since error will be continuously written to log above anyway. Removed it